Defs/ThingDefs_Misc/Apparel_Headgear.xml

```diff
@@ -176,4 +176,123 @@
+      </skillRequirements>
+        <researchPrerequisite>MicroelectronicsBasics</researchPrerequisite>      
+    </recipeMaker>
+    <costList>
+      <ComponentIndustrial>6</ComponentIndustrial>
+      <Steel>50</Steel>
+    </costList>
+        <statBases>
+            <WorkToMake>8000</WorkToMake>
+            <MaxHitPoints>100</MaxHitPoints>
+            <Mass>0.5</Mass>
+            <Bulk>3</Bulk>
+            <WornBulk>1</WornBulk>
+            <EquipDelay>0.5</EquipDelay>
+        </statBases>
+        <equippedStatOffsets>
+            <NightVisionEfficiency_Apparel>0.4</NightVisionEfficiency_Apparel>
+        </equippedStatOffsets>
+        <apparel>
+            <bodyPartGroups>
+                <li>Eyes</li>
+            </bodyPartGroups>
+            <wornGraphicPath>Things/Apparel/SimpleNightVision/SimpleNightVision</wornGraphicPath>
+            <layers>
+                <li>StrappedHead</li>
+            </layers>
+            <tags>
+                <li>IndustrialMilitaryAdvanced</li>
+            </tags>
+            <defaultOutfitTags>
+                <li>Soldier</li>
+            </defaultOutfitTags>
+              <hatRenderedFrontOfFace>false</hatRenderedFrontOfFace>
+        </apparel>
+        <modExtensions>
+            <li Class="CombatExtended.ApperalRenderingExtension">
+                <HideHair>false</HideHair>
+                <HideBeard>false</HideBeard>
+            </li>
+        </modExtensions>
+        <thingCategories Inherit="false">
+            <li>Headgear</li>
+        </thingCategories>
+    </ThingDef>
+
+    <ThingDef ParentName="ArmorHelmetMakeableBase">
+        <defName>Apparel_AdvancedNightVision</defName>
+        <label>advanced night vision goggles</label>
+        <description>An set of advanced night vision goggles offering detailed, panoramic vision when fighting in the dark.</description>
+        <graphicData>
+            <texPath>Things/Apparel/AdvancedNightVision/AdvancedNightVision_south</texPath>
+            <graphicClass>Graphic_Single</graphicClass>
+        </graphicData>
+    <techLevel>Industrial</techLevel>
+    <generateAllowChance>0.1</generateAllowChance>
+    <recipeMaker>
+      <unfinishedThingDef>UnfinishedTechArmor</unfinishedThingDef>
+      <skillRequirements>
+        <Crafting>9</Crafting>
+      </skillRequirements>
+      <researchPrerequisite>MultiAnalyzer</researchPrerequisite>         
+    </recipeMaker>
+    <costList>
+      <ComponentIndustrial>12</ComponentIndustrial>
+      <Steel>50</Steel>
+      <Plasteel>20</Plasteel>
+    </costList>
+        <statBases>
+            <WorkToMake>14000</WorkToMake>
+            <MaxHitPoints>100</MaxHitPoints>
+            <Mass>0.75</Mass>
+            <Bulk>5</Bulk>
+            <WornBulk>1</WornBulk>
+            <EquipDelay>0.5</EquipDelay>
+        </statBases>
+        <equippedStatOffsets>
+            <NightVisionEfficiency_Apparel>0.6</NightVisionEfficiency_Apparel>
+        </equippedStatOffsets>
```

Comment:
Same as above.

Reply:
We should set this to about `0.7`

Reply:
Fixed.

I set it to 0.6 because this is easier to produce than power armor, but has NV matching the basic power armor helmet.